appcues/build.gradle

```diff
@@ -124,3 +129,35 @@ dependencies {
     testImplementation 'androidx.arch.core:core-testing:2.2.0'
     testImplementation 'app.cash.turbine:turbine:1.0.0'
 }
+
+mavenPublishing {
```

Comment:
for some reason, it seems this block _must_ be in this build.gradle file, not applied from another file like we did before. Thus, some of the common property usage was split out in the `appcues-properties.grandle` and used here to keep the additions as minimal as possible. The old `publish-maven.gradle` is then no longer needed at all.